commit edbbbe099bf4d4902f29d087239d6d159ac2187d
Author: Kinglong Mee <kinglongmee@gmail.com>
Date:   Fri Apr 25 10:28:47 2014 -0400

    systemd: add PATH for finding systemctl
    
    The 1.3.0 release adds a call to systemctl fails for it's in /usr/bin.
    
    [root@localhost nfs-utils]# start-statd
    /usr/sbin/start-statd: line 9: systemctl: command not found
    Statd service already running!
    
    Reported-by: Allan Duncan <amd1234@fastmail.com.au>
    Signed-off-by: Kinglong Mee <kinglongmee@gmail.com>
    Signed-off-by: Steve Dickson <steved@redhat.com>

diff --git a/utils/statd/start-statd b/utils/statd/start-statd
index cde3583..8ac3798 100644
--- a/utils/statd/start-statd
+++ b/utils/statd/start-statd
@@ -4,7 +4,7 @@
 # /var/run/rpc.statd.pid).
 # It should run statd with whatever flags are apropriate for this
 # site.
-PATH=/sbin:/usr/sbin
+PATH="/sbin:/usr/sbin:/bin:/usr/bin"
 if systemctl start statd.service
 then :
 else
